mu4/notation/inotationinteraction.h

```diff
@@ -52,6 +52,7 @@ class INotationInteraction
 
     // Select
     virtual Element* hitElement(const QPointF& pos, float width) const = 0;
+    virtual int hitStaffIndex(const QPointF& pos) const = 0;
```

Comment:
I don't like to have this method in the interface. We've a pretty abstract method "hitElement" which should provide you a proper element which can provide you all necessary information. Otherwise, there will be A LOT of similar methods with different types of elements. I guess your "hitStaff" should be a private one in notationinteraction

Reply:
see https://github.com/musescore/MuseScore/pull/6714#discussion_r512454129